backends/jwt_javascript.go

```diff
@@ -90,14 +96,21 @@ func (o *jsJWTChecker) GetUser(token string) (bool, error) {
 	}
 
 	if o.options.parseToken {
-		username, err := getUsernameForToken(o.options, token, o.options.skipUserExpiration)
+		claims, err := getClaimsForToken(o.options, token, o.options.skipUserExpiration)
 
 		if err != nil {
-			log.Printf("jwt get user error: %s", err)
+			log.Printf("jwt get claims error: %s", err)
 			return false, err
 		}
 
-		params["username"] = username
+		if o.passClaims {
+			params["claims"] = claims
+		}
+		if username, found := claims[o.options.userFieldKey]; found {
+			params["username"] = username.(string)
+		} else {
+			params["username"] = ""
+		}
```

Comment:
This is used in the 3 checking methods, could probably be extracted to its own function/method.

Reply:
Fixed